designer/client/src/components/graph/node-modal/fragment-input-definition/item/Item.tsx

```diff
@@ -26,6 +26,12 @@ interface ItemProps {
     errors: NodeValidationError[];
 }
 
+//This projection is used for backward-compatibility reasons, since previously fragment input definition type options display part contained full class name
+function resolveSimpleClassName(className: string): string {
```

Comment:
could you contain `refClazzName` in the name of this function?, without digging implementation i had no idea that it's about  `refClazzName` and not react `className`